libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -2542,13 +2540,8 @@ void IRGeneratorForStatements::appendBareCall(
 	}
 
 	templ("success", IRVariable(_functionCall).tupleComponent(0).name());
-	if (IRVariable(_functionCall).tupleComponent(1).type().category() == Type::Category::InaccessibleDynamic)
-		templ("returndataVar", "");
-	else
-	{
-		templ("returndataVar", IRVariable(_functionCall).tupleComponent(1).part("mpos").name());
-		templ("extractReturndataFunction", m_utils.extractReturndataFunction());
-	}
```

Comment:
Why has the distinction between inaccessiblydynamic return types been removed?

Reply:
Because `m_utils.extractReturndataFunction` already handles both cases.